code/modules/jobs/job_types/agent.dm

```diff
@@ -22,7 +22,57 @@
 
 	var/normal_attribute_level = 20 // Scales with round time
 
-/datum/job/agent/after_spawn(mob/living/H, mob/M, latejoin = FALSE)
+/datum/job/agent/after_spawn(mob/living/carbon/human/H, mob/M, latejoin = FALSE)
+	// Assign department security
+	var/department
+	if(M && M.client && M.client.prefs)
+		department = M.client.prefs.prefered_agent_department
+		if(department == "None")
+			return
+	var/ears = null
+	var/accessory = null
+	switch(department)
+		if("Control")
+			ears = /obj/item/radio/headset/headset_control
+			accessory = /obj/item/clothing/accessory/armband/lobotomy
+		if("Command")
+			ears = /obj/item/radio/headset/headset_command
```

Comment:
Command department is still used for command things. I don't recommend giving them this headset.